README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img alt="Differencify" src="http://i.imgur.com/D0Eapjx.png" width="150">
+<p align="center"><img alt="Differencify" src="http://i.imgur.com/0zwMggK.png" width="150">
```

Comment:
If it's something the repo needs, it should be in the repo. Would you put Api.md on pastebin (a bit of an exaggeration but you get the idea)?